keras/wrappers/scikit_learn.py

```diff
@@ -6,6 +6,18 @@
 from ..utils.np_utils import to_categorical
 
 
+class Hideout(object):
```

Comment:
Since this is only for use by one method in one class, you should define this class in the scope of that method.

Additionally, it doesn't seem (intuitively) to be a very clean pattern. Are you sure this is the best way to proceed?